packages/s/slim.js.json

```
@@ -15,13 +15,13 @@
     "binding"
   ],
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/eavichay/slim.js.git",
+    "source": "npm",
+    "target": "slim.js",
```

```suggestion
    "target": "slim-js",
```
`slim.js` is absolutely not the same package, we need `slim-js`, per https://github.com/slimjs/slim.js/blob/master/package.json

---

Whoops, good catch :)